structuralcodes/sections/section_integrators/_fiber_integrator.py

```diff
@@ -228,9 +249,115 @@ def integrate_strain_response_on_geometry(
             Mx and My and the triangulation data.
         """
         # Prepare the general input based on the geometry and the input strains
-        prepared_input, triangulated_data = self.prepare_input(
+        prepared_input, triangulated_data = self.prepare_input_stress(
+            geo, strain, **kwargs
+        )
+
+        # Return the calculated response
+        return *self.integrate_stress(prepared_input), triangulated_data
+
+    def prepare_input_tangent(
```

Comment:
To this and the other related methods. Do we anticipate that we will only include the tangent stiffness, and not a secant or initial etc.? If we anticipate to expand later, I think we should avoid calling the method `_tangent`.